net-wireless/hostapd/hostapd-2.9-r6.ebuild

```diff
@@ -193,7 +194,11 @@ src_configure() {
 	# Removed for now, since the 3.2 version is broken, and we don't
 	# support it.
 	if has_version ">=dev-libs/libnl-3.2"; then
+		local libnl_include
+
 		echo "CONFIG_LIBNL32=y" >> ${CONFIG} || die
+		libnl_include=$($(tc-getPKG_CONFIG) --cflags libnl-3.0)
+		echo "CFLAGS += ${libnl_include}" >> ${CONFIG} || die
```

Comment:
I would've thought we could just do append-cflags?

Reply:
Sure, let me just test it quickly

Reply:
Works perfectly :)

Reply:
Thanks for checking so quickly!

Just so you can internalise the thought process: the idea was, we're modifying CFLAGS, but surely we respect CFLAGS in this package, so let's append it in the normal way and see what breaks. That way, we could just fix the "real" problem (not respecting CFLAGS) if necessary. But tbh, this is one of those packages where the build system is scary and it messes wih your head.